Hmpv blog (#1101)
nextstrain/nextstrain.org: static-site/content/blog/2025-01-06-human-metapneumovirus-analysis-and-resources.md

---
author: "Isabel Joia, Richard Neher"

> date: "2025-01-05"

Reviewer: Small reminder that the date should be updated before merging. Merging will automatically put it on https://next.nextstrain.org. I (or another dev) can then push it to https://nextstrain.org on the same day.

```suggestion
date: "2025-01-06"
```

Reviewer: #1103

title: "Phylogenetic analysis of Human Metapneumovirus"

> sidebarTitle: "Human Metapneumovirus Phylogenetic Analysis and Resources"

Reviewer 2: Perhaps to save some space? 

```suggestion
sidebarTitle: "HMPV Phylogenetic Analysis and Resources"
```

Author: Thank you. I've implemented the changes.

---

> We are pleased to announce the availability of the hMPV phylogenetic analysis on [Nextstrain](https://nextstrain.org) and its corresponding [Nextclade dataset](https://docs.nextstrain.org/projects/nextclade/en/stable/user/datasets.html). Below describes an overview of hMPV and its phylogeny. We welcome feedback from hMPV researchers about any aspects of this analysis and dataset.

Reviewer 2: I suggest linking to the dataset itself, rather than the docs; also, phrasing suggestion

```suggestion
We are pleased to announce the availability of the hMPV phylogenetic analysis on [Nextstrain](https://nextstrain.org) and its [corresponding Nextclade dataset](nextstrain.org/fetch/data.clades.nextstrain.org/v3/nextstrain/hmpv/all-clades/NC_039199/2024-12-04--17-05-31Z/tree.json). Below, we describe an overview of hMPV and its phylogeny. We welcome feedback from hMPV researchers about any aspects of this analysis and dataset.
```

# Sequencing Data

The hMPV genome consists of 8 different genes, the most commonly sequenced ones being gene F and gene G, which encode surface proteins, more specifically, the fusion protein and the glycoprotein. On [nextstrain.org/hmpv](https://nextstrain.org/hmpv) we host the phylogenies of clades A and B based of the following sequences:

1. Full genome
2. Gene G
3. Gene F

The phylogenies are generated using genomic data from [NCBI GenBank](https://www.ncbi.nlm.nih.gov/), and are updated when new sequences are uploaded to NCBI. Currently, over 11,000 sequences are available, over 800 sequences for the entire genome, 1800 sequences for gene F and 3200 sequences for gene G. We curate sequence data and metadata from NCBI as the starting point for our analyses. 

![fig2](/blog/img/hmpv_a_G.png)
**Figure 2. Global distribution and time resolved phylogeny of clade A using sequences from gene G.** More extensive and representative phylogenies for gene G and gene F are possible, as more genetic data is available for these genes. Recently, we have seen a global increase in prevalence of clade A2b2 (in orange).

# Nextclade Dataset

> As part of this work, we developed an [open-source tool for classifying hMPV genotypes](https://clades.nextstrain.org/). This tool is a [Nextclade dataset]((https://docs.nextstrain.org/projects/nextclade/en/stable/user/datasets.html)), which allows users to drag-and-drop sequences onto a web browser to obtain the alignment to a hMPV reference genome from clade A. It also outputs quality control metrics, calls mutations, assigns clades and displays user sequences on the phylogenetic tree (**Fig. 3**). 

Reviewer 3: The link to the Nextclade docs didn't work for me, which I suspect is because of the extra parentheses here.

```suggestion
As part of this work, we developed an [open-source tool for classifying hMPV genotypes](https://clades.nextstrain.org/). This tool is a [Nextclade dataset](https://docs.nextstrain.org/projects/nextclade/en/stable/user/datasets.html), which allows users to drag-and-drop sequences onto a web browser to obtain the alignment to a hMPV reference genome from clade A. It also outputs quality control metrics, calls mutations, assigns clades and displays user sequences on the phylogenetic tree (**Fig. 3**). 
```

Author: Thank you for the suggestion.

> As part of this work, we developed an [open-source tool for classifying hMPV genotypes](https://clades.nextstrain.org/). This tool is a [Nextclade dataset]((https://docs.nextstrain.org/projects/nextclade/en/stable/user/datasets.html)), which allows users to drag-and-drop sequences onto a web browser to obtain the alignment to a hMPV reference genome from clade A. It also outputs quality control metrics, calls mutations, assigns clades and displays user sequences on the phylogenetic tree (**Fig. 3**). 

Reviewer 3: I suggest using a link that goes directly to the hmpv nextclade dataset

```suggestion
As part of this work, we developed an [open-source tool for classifying hMPV genotypes](https://clades.nextstrain.org/?dataset-name=nextstrain/hmpv/all-clades/NC_039199). This tool is a [Nextclade dataset]((https://docs.nextstrain.org/projects/nextclade/en/stable/user/datasets.html)), which allows users to drag-and-drop sequences onto a web browser to obtain the alignment to a hMPV reference genome from clade A. It also outputs quality control metrics, calls mutations, assigns clades and displays user sequences on the phylogenetic tree (**Fig. 3**). 
```

Author: Thank you.

![fig3](/blog/img/hmpv_nextclade.png)
**Figure 3. Example output for the hMPV Nextclade dataset.** Each row contains the analysis for one sequence and includes (from left to right) the sequence name, quality control metrics, clade assignment, number of mutations, number of ambiguous nucleotides, number of missing nucleotides, coverage of the sequence, gaps, insertions, frame shifts, premature stop codons and mutation placement. 

# Duplications in Gene G

> While historically, clades A and B have annually circulated simultaneously to a similar extent, an increase in prevalence of A2b2 strains over the last few years has been observed. A common characteristic in these strains are up to 180 nucleotide long [duplications in the G gene](https://pmc.ncbi.nlm.nih.gov/articles/PMC5348506/), which seem to have emerged around 2011. Interestingly enough, a similar evolution took place in RSV, where two independent duplications in the G gene occured and strains with these duplications quickly became the most dominant RSV subtypes. Both in RSV and hMPV, the duplications in the G gene seem to confer an evolutionary advantage. 

Reviewer 3: > Interestingly enough, a similar evolution took place in RSV,

Is there a reference you could provide for this?
